tests/migration_test.go

```diff
@@ -1571,9 +1571,32 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 					targetNode.Name,
 					targetNode.Name,
 				))
+
+				By("Checking that all migrated VMIs have the new pod IP address")
+				for _, vmi := range vmis {
+					Eventually(func() error {
+						newvmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
+						Expect(err).ToNot(HaveOccurred())
+						vmiPod := tests.GetRunningPodByVirtualMachineInstance(newvmi, tests.NamespaceTestDefault)
+
+						for _, network := range newvmi.Spec.Networks {
+							if network.NetworkSource.Pod != nil {
```

Comment:
sorry in advance if i mistaken,

the code has a break 
"// Currently will only traverse to the first Pod network found"
the test doesnt in case there is more than one pod network found
is it intentional or need to add a break on the first one found ?